src/main/java/study3/controller/LadderController.java

```diff
@@ -0,0 +1,46 @@
+package study3.controller;
+
+import java.util.List;
+
+import study3.domain.ExecuteResult;
+import study3.domain.Height;
+import study3.domain.Ladder;
+import study3.domain.Players;
+import study3.domain.Results;
+import study3.domain.Rewards;
+import study3.view.InputView;
+import study3.view.ResultView;
+
+public class LadderController {
+	private static final String WANT_ALL = "all";
+
+	public static void main(String[] args) {
+
+		InputView inputview = new InputView();
+
+		Players players = new Players(inputview.getParticipantNames());
+
+		ExecuteResult executeResult = new ExecuteResult(inputview.getExecuteResult());
+
+		Height height = new Height(inputview.inputHeight());
+
+		Ladder ladder = new Ladder(players, height);
```

Comment:
아래에서 불필요하게 사용하는 `players` 등의 참조를 모두 제거한다고 하면

조금 길어지겠지만 아래와 같이 변경하는건 어떨까요?

```suggestion
		Ladder ladder = new Ladder(new Players(inputview.getParticipantNames()), new Height(inputview.inputHeight()));
```

추가로 생성자 대신 정적 팩토리 메소드 등을 활용할수도 있을 것 같습니다.

Reply:
이부분은 executeResult 가 players와 height 사이에서 입력을 받아야 해서 수정이 불가능할거 같은데 맞을까요? ㅠ

----------------------------------------------------------------------------------
참여할 사람 이름을 입력하세요. (이름은 쉼표(,)로 구분하세요)
pobi,honux,crong,jk

실행 결과를 입력하세요. (결과는 쉼표(,)로 구분하세요)
꽝,5000,꽝,3000

최대 사다리 높이는 몇 개인가요?
5